docs/topics/auth/customizing.txt

```diff
@@ -299,6 +299,35 @@ example, the following checks if a user may view tasks::
 
     user.has_perm('app.view_task')
 
+Custom permissions may contain the ``%(class)s`` and ``%(verbose_name)s``
+string. Child models will then inherit the specific permission::
```

Comment:
string -> placeholders
The documentation might suggest that this can only be used on abstract models but I don't think that's the case, is it?